OneCommunityGlobal/HGNRest
# Haoyue fix role title

# Description

![Screenshot 2024-09-20 at 1 22 32 PM](https://github.com/user-attachments/assets/93585d96-b166-494d-abee-0cd968fdada6)

## Related PRS (if any):
To test this backend PR you need to checkout the [#2700](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2700) frontend PR.
…

## Main changes explained:
- Added the validation for Title Code field.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. follow the steps in the front end PR.

## Screenshots or videos of changes:


https://github.com/user-attachments/assets/4ffbc7b3-7e89-4f90-95f4-738412066186


## Note:
Include the information the reviewers need to know.
